src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,73 @@
+package core.basesyntax.model;
+
+public class FruitTransaction {
+    private Operation operation;
+    private String fruit;
+    private int quantity;
+
+    public FruitTransaction(Operation operation, String fruit, int quantity) {
+        this.operation = operation;
+        this.fruit = fruit;
+        this.quantity = quantity;
+    }
+
+    public Operation getOperation() {
+        return operation;
+    }
+
+    public void setOperation(Operation operation) {
+        this.operation = operation;
+    }
+
+    public String getFruit() {
+        return fruit;
+    }
+
+    public void setFruit(String fruit) {
+        this.fruit = fruit;
+    }
+
+    public int getQuantity() {
+        return quantity;
+    }
+
+    public void setQuantity(int quantity) {
+        this.quantity = quantity;
+    }
+
+    @Override
+    public String toString() {
+        return "FruitTransaction{"
+                + "operation=" + operation
+                + ", fruit='" + fruit
+                + '\'' + ", quantity=" + quantity
+                + '}';
+    }
+
+    public enum Operation {
```

Comment:
The `Operation` enum is a part of the `FruitTransaction` class, which is a good use of the encapsulation principle. However, it is recommended to place enums in a separate file if they are used by multiple classes. If `Operation` is used outside of `FruitTransaction`, consider moving it to its own file.